conjure-codegen/src/example_types/product/safe_long_example.rs

```diff
@@ -2,74 +2,23 @@ use conjure_object::serde::{ser, de};
 use conjure_object::serde::ser::SerializeStruct as SerializeStruct_;
 use std::fmt;
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Copy)]
+#[conjure_object::private::staged_builder::staged_builder]
+#[builder(crate = conjure_object::private::staged_builder, update, inline)]
 pub struct SafeLongExample {
+    #[builder()]
```

Comment:
I notice that most of the `conjure_object` types don't need to specify `into` here. What makes that possible, whereas the `String` type does need the `into`?

Reply:
It depends on what conversion behavior we want to support for the setter. With an e.g. `i32` we just take that type directly, whereas for string we take `impl Into<String>` so you can pass in e.g. string literals directly.

Reply:
The empty `#[builder()]` attributes aren't necessary - I pushed up a commit to remove them.